### PR TITLE
216 mock account related functions

### PR DESCRIPTION
**Pull request by Elli610**

### CODE REVIEW

**Review by 0xSwapFeeder — APPROVED** [review]
lgtm

File: vm-mock/vm.js [hunk 1]
```diff
@@ -376,6 +376,15 @@ export default function createMockedABI(
       assembly_script_get_call_coins() {
         return BigInt(0);
       },
+
+      assembly_script_signature_verify(digestPtr, signaturePtr, publicKeyPtr) {
+        return true;
+      },
+
+      assembly_script_address_from_public_key(publicKeyPtr) {
+        return newString("AU12UBnqTHDQALpocVBnkPNy7y5CndUJQTLutaVDDFgMJcq5kQiKq");
```
[the remark this answers is not in the document]
Comment (Ben-Rey) [reply vm-mock/vm.js:385]: It might not be so easy to implement.

[the remark this answers is not in the document]
Comment (Ben-Rey) [reply vm-mock/vm.js:385]: @gregLibert do you think it's important to have real mock for this kind of functions ?

[the remark this answers is not in the document]
Comment (gregLibert) [reply vm-mock/vm.js:385]: it's not a priority, so you should create an "good first" kind issue explaining that we can improve this.

From a general perspective, you have two ways of solving this:
- dynamic calculation
- creating a setter to let the tester have the expected value using it.

Ping me if this is not clear.

[the remark this answers is not in the document]
Comment (Ben-Rey) [reply vm-mock/vm.js:385]: very clear thanks

**Review by Ben-Rey — APPROVED** [review]

**Comment by Ben-Rey** [comment]
@gregLibert we can merge if you agree

**Review by gregLibert — APPROVED** [review]

**Pull request merged**